api/db/populate-friendships.js

```diff
@@ -0,0 +1,85 @@
+const { getRandomSubset, getRandomDate, createPrompt } = require('./utils');
+
+const User = require('../models/User');
+const Friendship = require('../models/Friendship');
+
+let displayLogs;
+
+const createFriendship = async (friendshipParams) => {
+  const friendship = new Friendship({
+    requestant: friendshipParams.requestant,
+    receivant: friendshipParams.receivant,
+    accepted: friendshipParams.accepted,
+    acceptedAt: friendshipParams.acceptedAt,
+    lastMessageAt: friendshipParams.lastMessageAt,
+  });
+  await friendship.save();
+
+  if (displayLogs) {
+    console.log('\n', '\x1b[0m', `New friendship created: ${friendship}`);
+  }
+  return friendship;
+};
+
+const createFriendships = async (verbose) => {
+  console.log('\n', '\x1b[0m', 'Populating friendships collection...');
+  displayLogs = verbose;
+
+  let usersList = await User.distinct('_id');
+  const friendshipsArray = await Friendship.find();
```

Comment:
For consistency, shouldn't this be named `friendshipsList`?

Reply:
Yes and no. In order to populate the friendships collection, I need to get the friendships collection that already exists. And then as I create the friendships, I add them to this array so that I can check to see if the friendship exists in the next iteration. This should answer your last comment below as well.

Reply:
Also, for all the other scripts it might be worth considering getting rid of the <collection>Array anyway. We don't use it for anything other console logging the amount of records generated, and that could just as easily be changed to an integer variable that we update whenever a record is saved to the db.

Reply:
Aha, that makes sense. And your second point makes sense too! I would say that, after all scripts are added, we should submit a follow up PR addressing this and other small issues that we might find to finally close #279.